.github/workflows/ci.yml

```diff
@@ -20,7 +20,14 @@ jobs:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v2
-    - name: Format
+    - name: Restore cache
+      uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-    - name: Format
```

Comment:
Looks like there might've been a copy-paste error here with the trailing `- name: Format`. Could you remove that?

Reply:
Hey Ben! It was a copy-paste error, in that I replaced line 23 instead of inserting above it. The original file has the "Format" step on lines 23-25. It has been transposed to lines 30-32 in this version.

Reply:
Hey @reeseconor, I think Ben is pointing out this specific line:  `${{ runner.os }}-go-    - name: Format`

Reply:
Yup, probably should've used a suggestion since it's easier to see on GitHub, so here's one below.
```suggestion
          ${{ runner.os }}-go-
```